src/Algebra/Construct/Add/Identity.agda

```diff
@@ -0,0 +1,103 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of algebraic structures we get from freely adding an
+-- identity element
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Add.Identity where
+
+open import Algebra.Bundles
+open import Algebra.Core using (Op₂)
+open import Algebra.Definitions
+open import Algebra.Structures
+open import Data.Maybe.Relation.Binary.Pointwise
```

Comment:
Hmm I've just realised that we should really be using

https://github.com/agda/agda-stdlib/blob/650e05f3a5ef06cb938c3b91a4fc8b9b8d7c5ef2/src/Relation/Binary/Construct/Add/Point/Equality.agda#L27

as the underlying equality rather than the definition from `Data.Maybe.Relation.Binary.Pointwise`.

I can't remember why the former isn't implemented using the latter... It probably should be, but that's for another issue.